src/main/server/player/server-player-component.ts

```diff
@@ -40,6 +41,7 @@ export class ServerPlayerComponent {
          },
          moving: false,
          cheese: 0.0,
+         type: MathUtil.randomIntFromInterval(0, ServerConfig.NR_OF_PLAYER_TYPES),
```

Comment:
I have created a array of all players with their sprites in a nice array in another branch

Reply:
Lets merge yours first and I'll change it to use that